src/state/reducer.js

```diff
@@ -107,17 +107,25 @@ const move = ({
     windowScroll: currentWindowScroll,
   };
 
+  const previousDroppableOver = state.drag && state.drag.impact.destination
+    ? state.drag.impact.destination.droppableId
+    : null;
+
   const newImpact: DragImpact = (impact || getDragImpact({
     pageCenter: page.center,
     draggable: state.dimension.draggable[current.id],
     draggables: state.dimension.draggable,
     droppables: state.dimension.droppable,
+    previousDroppableOver,
   }));
 
   const drag: DragState = {
     initial,
     impact: newImpact,
     current,
+    previous: {
+      droppableOver: previousDroppableOver,
```

Comment:
can this just be the `id` and we keep the dimension within the `state.dimension`?

Reply:
Oh sorry, I just saw it was the Id. Can this be changed to `droppableOverId`